### PR TITLE
Remove nsp and fix vulnerabilities

### PR DESCRIPTION
**Pull request by antleblanc**
# Remove nsp and fix vulnerabilities

## 🐛 Bug Fix

de8d57c - fix(deps): remove nsp and fix vulnerabilities

- Prepublish hook now runs: `npm audit`.
- `yeoman-generator` has been upgraded to `v4.2.0`.
- Both `jest` and `jest-cli` has been upgraded to `v25.0.0`.
- `package-lock.json` has been added (needed for `npm audit` command).

## 🔗 Related

- https://www.npmjs.com/package/nsp
- https://blog.npmjs.org/post/175511531085/the-node-security-platform-service-is-shutting

## 📝 Documentation

c59a637 - docs(contributing): remove extra requirement regarding translations

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -8,7 +8,6 @@ pay attention to a few things:
 3. your code must be documented
 4. your work must be signed (see below)
 5. you may contribute through GitHub Pull Requests on `master` branch
-6. the sentences must be done in French (temporarily, due to internal process)
```
Comment (FredericEspiau) [inline .github/CONTRIBUTING.md]: Why did you remove this ?

Comment (antleblanc) [reply .github/CONTRIBUTING.md]: This generator doesn't rely on any translations files.

I suspect a copy/paste from previous `ovh-ux/ovh-manager-*` repositories.

**Comment by blaryjp** [comment]
tx al
